### PR TITLE
Remove uncalled configureSysInit

### PR DESCRIPTION
**Pull request by lowenna**
Signed-off-by: John Howard jhoward@microsoft.com

Was scanning for "TODO Windows" and noticed configureSysInit() is no longer called from anywhere in the docker code. Removed.

### CODE REVIEW

File: daemon/daemon_windows.go [hunk 1]
```diff
@@ -91,10 +90,10 @@ func migrateIfDownlevel(driver graphdriver.Driver, root string) error {
 	return nil
 }
 
-func configureSysInit(config *Config, rootUID, rootGID int) (string, error) {
-	// TODO Windows.
-	return os.Getenv("TEMP"), nil
-}
+//func configureSysInit(config *Config, rootUID, rootGID int) (string, error) {
```
Comment (LK4D4) [inline daemon/daemon_windows.go]: Hmm, why not just remove it? I don't like commented code.

Comment (jhowardmsft) [inline daemon/daemon_windows.go]: Oh I meant to. I just commented out to confirm that it still compiled on Windows, forgot to actually remove it subsequently. Will push in a minute.

Comment (jhowardmsft) [inline daemon/daemon_windows.go]: Updated. Sorry about that. Good catch :smile:

**Comment by LK4D4** [comment]
Removing dockerversion import here is cool :)

**Comment by LK4D4** [comment]
LGTM

**Pull request merged**